### PR TITLE
add analytics to docs

### PR DESCRIPTION
**Pull request by OriolAbril**
Adds google analytics to the documentation page. We'll then gather some data now and after season of
docs we'll be able to see if there is an increase in traffic and retention of visitors.

### CODE REVIEW

File: docs/make.jl
Comment (sethaxen) [inline docs/make.jl:23]: The formatter prefers no whitespace here:

```suggestion
    analytics="G-W1G68W77YV",
```

**Review by sethaxen — APPROVED** [review]
LGTM, once the formatter workflow passes

**Comment by OriolAbril** [comment]
Looks like we'll have the same issue here. I'll try to send a PR to update https://github.com/JuliaDocs/Documenter.jl/blob/master/src/Writers/HTMLWriter.jl#L891 and work with either `G-` or `UA-` codes.

**Comment by OriolAbril** [comment]
It looks like there is an error with the netcdf roundtrip and with summarystats. Are these tests on master or latest version?

**Comment by sethaxen** [comment]
> Looks like we'll have the same issue here. I'll try to send a PR to update https://github.com/JuliaDocs/Documenter.jl/blob/master/src/Writers/HTMLWriter.jl#L891 and work with either `G-` or `UA-` codes.

Is https://github.com/JuliaDocs/Documenter.jl/pull/1559 relevant?

> It looks like there is an error with the netcdf roundtrip and with summarystats. Are these tests on master or latest version?

These are in the Futures workflow, which is allowed to fail and just lets us know of future breaking changes (from unreleased Jula or ArviZ versions). They shouldn't hold up this PR. The netcdf one I haven't seen before and will look into.

**Comment by sethaxen** [comment]
The docs build is timing out after 6 hours though. In the past, this has happened due to some mutual incompatibility of some dependency(s) of Soss, Turing, and Documenter, and I've never been able to diagnose it; it usually gets resolved when some version is released.

Pretty much the only way around this besides waiting for it to be fixed elsewhere is probably to split the Quickstart into different pages for Soss, Turing, and CmdStan and execute the examples in their own environment.

**Comment by OriolAbril** [comment]
> Is JuliaDocs/Documenter.jl#1559 relevant?

Indeed, this will fix the issue :). Feel free to merge whenever is best for you, analytics won't work until that PR is merged but in the meantime, this should not break docs nor have any noticeable effect in them, it's only a cryptic javascript error that you can only see if you open the web developer tools.

**Pull request merged**
